vertical-pod-autoscaler/e2e/v1/common.go

```diff
@@ -297,14 +297,15 @@ func SetupVPA(f *framework.Framework, cpu string, mode vpa_types.UpdateMode, tar
 
 // SetupVPAForNHamsters creates and installs a simple hamster VPA for a pod with n containers, for e2e test purposes.
 func SetupVPAForNHamsters(f *framework.Framework, n int, cpu string, mode vpa_types.UpdateMode, targetRef *autoscaling.CrossVersionObjectReference) *vpa_types.VerticalPodAutoscaler {
-	return SetupVPAForNHamstersWithMinReplicas(f, n, cpu, mode, targetRef, nil)
+	return SetupVPAForNHamstersWithMinReplicas(f, n, cpu, mode, targetRef, nil, nil)
 }
 
 // SetupVPAForNHamstersWithMinReplicas creates and installs a simple hamster VPA for a pod with n containers, setting MinReplicas. To be used for e2e test purposes.
-func SetupVPAForNHamstersWithMinReplicas(f *framework.Framework, n int, cpu string, mode vpa_types.UpdateMode, targetRef *autoscaling.CrossVersionObjectReference, minReplicas *int32) *vpa_types.VerticalPodAutoscaler {
+func SetupVPAForNHamstersWithMinReplicas(f *framework.Framework, n int, cpu string, mode vpa_types.UpdateMode, targetRef *autoscaling.CrossVersionObjectReference, minReplicas *int32, er []*vpa_types.EvictionRequirement) *vpa_types.VerticalPodAutoscaler {
```

Comment:
Seems like we're never setting both minReplicas and er; Please:

- Add a new helper

or even better

- Add a builder (which will be cleaner to use)

Reply:
Thanks, that makes sense! I'll make the builders available in the e2e tests and push an update soon.

Reply:
done.